Fritz.StreamTools.sln

```diff
@@ -3,10 +3,6 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27130.2020
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{E53339B2-1760-4266-BCC7-CA923CBCF16C}") = "docker-compose", "docker-compose.dcproj", "{520B1405-AC30-493A-8F04-6AA9B8FE2CDD}"
```

Comment:
Careful. It looks like we've removed the docker-compose and test projects.

Reply:
Hmm i also noticed that i don't have test project in same solution... :(
I think i broke it i'll investigate